OpenRA.Game/Map/IMap.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System;
+using System.Collections.Generic;
+using OpenRA.Graphics;
+using OpenRA.Primitives;
+using OpenRA.Support;
+using OpenRA.Traits;
+
+namespace OpenRA
+{
+	public interface IMap : IDisposable
+	{
+		Dictionary<CPos, TerrainTile> ReplacedInvalidTerrainTiles { get; }
+		MapGrid Grid { get; }
+		int2 MapSize { get; }
+		string Tileset { get; }
+		Ruleset Rules { get; }
+		SequenceSet Sequences { get; }
+		CellRegion AllCells { get; }
+		List<CPos> AllEdgeCells { get; }
+		WDist CellHeightStep { get; }
```

Comment:
Why isn't this on the Elecation interface?

Reply:
https://github.com/OpenRA/OpenRA/pull/20582#discussion_r1159809521